src/utils.tsx

```diff
@@ -49,3 +49,25 @@ export const nl2br = (text: string | null) => {
 export const formatFilesize = (size: number): string => {
   return filesize(size, { standard: 'iec' });
 };
+
+export const splitArrayIntoChunks = <ItemType extends {}>(
```

Comment:
Can we have a comment to indicate that `extends {}` is needed because TS does not understand that it's a generics otherwise?